cmd/kafka-consumer/main.go

```diff
@@ -502,8 +502,9 @@ func (c *Consumer) ConsumeClaim(session sarama.ConsumerGroupSession, claim saram
 						zap.Uint64("ts", ts),
 						zap.Uint64("resolvedTs", resolvedTs),
 						zap.Int32("partition", partition))
-				} else if ts > resolvedTs {
-					log.Debug("update sink resolved ts",
+				}
+				if ts > resolvedTs {
```

Comment:
so, here no logic is changed, only change the log level?

Reply:
yes, for better observe the resolved ts update logic.